### ISSUE TITLE
Add support for multiple --platform and --abi

### ISSUE THREAD

**Issue by shk3bq4d** [montag451/pypi-mirror#16]
Hello,

Thank you for your beautiful project, it is much appreciated!

from pip (21.3.1) install --help command platform and abi may be repeated but the pypi-mirror CLI parser doesn't

with pip I can execute a cross abi request like that:
```
pip install \
    --target=/tmp  \
    --only-binary=:all:  \
        cffi \
        cryptography  \
        --abi cp38  \
        --abi cp36 \
        --platform manylinux_2_12_x86_64 \
        --platform manylinux_2_24_x86_64 \
        --python-version 38 \
        --implementation cp

Collecting cffi
  Using cached cffi-1.15.0-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (446 kB)
Collecting cryptography
  Using cached cryptography-36.0.0-cp36-abi3-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (3.7 MB)
Collecting pycparser
  Using cached pycparser-2.21-py2.py3-none-any.whl (118 kB)
Installing collected packages: pycparser, cffi, cryptography
Successfully installed cffi-1.15.0 cryptography-36.0.0 pycparser-2.21
```

support pip install --no-deps would also help for installing packages one by one would be a workaround for such case.

Thank you and best regards!

**Comment by montag451** [comment]
Hi @shk3bq4d,

First of all, I wish you a happy new year. Secondly I apologize for my late reply I've been quite busy the past weeks. Allowing multiple --platform and --abi is indeed very useful and I've implemented it in 0671ca487e63c128e9b98cfac753c8b04e92afd3. If it's OK for you I will publish a new version on PyPI.

**Comment by montag451** [comment]
I published a new release on [PyPI](https://pypi.org/project/python-pypi-mirror/4.2.0/) which contain the new option you requested. Thank you for the issue report!
